Makefile

```diff
@@ -335,3 +336,22 @@ svgtest: ../owid-grapher-svgs
 ../owid-content:
 	@echo '==> Cloning owid-content to ../owid-content'
 	cd .. && git clone git@github.com:owid/owid-content
+
+node_modules: package.json yarn.lock yarn.config.cjs
+	@echo '==> Installing packages'
+	yarn install
+
+itsJustJavascript: node_modules
+	@echo '==> Compiling TS'
+	yarn lerna run build
+	yarn run tsc -b
+	touch $@
+
+reindex: itsJustJavascript
+	@echo '==> Reindexing search in Algolia'
+	node --enable-source-maps itsJustJavascript/baker/algolia/indexToAlgolia.js
```

Comment:
I guess we could also run `configureAlgolia` here?

It's also run as part of every deploy ([see here](https://github.com/owid/owid-grapher/blob/f401d1169ce5f2c9224d9d8f972052512081c428/ops/buildkite/build-code#L97)), and is a no-op if nothing was changed in `configureAlgolia`.

Reply:
Good catch! This actually configured and fixed the search index, it was the missing piece for me.